ansible/roles/ocp4-workload-ml-workflows-workshop/tasks/workload.yml

```diff
@@ -4,9 +4,24 @@
     msg: User count end({{ user_count_end }}) < user count start ({{ user_count_start }})
   when: user_count_end|int < user_count_start|int
 
+- name: Check that Tekton resources are defined
+  k8s_info:
+    api_version: apiextensions.k8s.io/v1
+    kind: CustomResourceDefinition
+    name: "{{ item }}"
+  loop:
+  - pipelineresources.tekton.dev
+  - pipelines.tekton.dev
```

Comment:
Should we also include `task.tekton.dev` as we are setting up a Task definition later too?

Reply:
Sounds good, I'll add that as well.